q2_feature_table/plugin_setup.py

```diff
@@ -338,7 +353,15 @@
     description="Filter samples from table based on frequency and/or "
                 "metadata. Any features with a frequency of zero after sample "
                 "filtering will also be removed. See the filtering tutorial "
-                "on https://docs.qiime2.org for additional details."
+                "on https://docs.qiime2.org for additional details.",
+    examples={
+        'filter_to_subject1': feature_table_filter_samples_metadata1,
+        'filter_to_skin': feature_table_filter_samples_metadata2,
+        'filter_to_subject1_gut': feature_table_filter_samples_metadata3,
+        'filter_to_gut_or_abx': feature_table_filter_samples_metadata4,
```

Comment:
These are good user-facing names - maybe just match the example names with the user facing naming conventions? Sorry, I've suggested like three different ways to change the names lol - I think this might be the best approach though.